client/signup/jetpack-connect/plans-landing.jsx

```diff
@@ -16,28 +16,28 @@ import QueryPlans from 'components/data/query-plans';
 
 const CALYPSO_JETPACK_CONNECT = '/jetpack/connect';
 
-const PlansLanding = React.createClass( {
+class PlansLanding extends Component {
 
-	propTypes: {
-		sites: React.PropTypes.object,
-		sitePlans: React.PropTypes.object.isRequired,
-		intervalType: React.PropTypes.string
-	},
+	static propTypes = {
+		basePlansPath: PropTypes.string,
```

Comment:
When adding non-required props, maybe you'll also want to add some `defaultProps` for them?